common/achievement/derive.ts

```diff
@@ -0,0 +1,156 @@
+import {
+    pupil as Pupil,
+    achievement_action_type_enum as AchievementActionType,
+    achievement_template_for_enum as AchievementTemplateFor,
+    achievement_type_enum as AchievementType,
+    achievement_template,
+    pupil_screening_status_enum,
+} from '@prisma/client';
+import { User, getPupil, getStudent } from '../user';
+// TODO: Fix import when other PR is in
+import { prisma } from '../prisma';
+import { achievement_with_template } from './types';
+import { getAchievementTemplates, TemplateSelectEnum } from './template';
+
+const GhostAchievements: { [key: string]: achievement_template } = {
+    pupil_new_match_1: {
+        id: -1,
+        templateFor: AchievementTemplateFor.Match,
+        group: 'pupil_new_match',
+        groupOrder: 1,
+        type: AchievementType.SEQUENTIAL,
+        image: 'gamification/achievements/release/finish_onboarding/two_pieces/step_1.png',
+        tagline: 'Starte eine Lernpatenschaft',
+        title: 'Neue Lernunterstützung',
+        subtitle: null,
+        description:
+            'Es war großartig, dich am {{date}} besser kennenzulernen und freuen uns, dass du gemeinsam mit uns die Bildungschancen von Schüler:innen verbessern möchtest. Um dir eine:n passende:n Lernpartner:in zuzuweisen, bitten wir dich zunächst, eine Anfrage auf unserer Plattform zu stellen. Hier kannst du die Fächer und Jahrgangsstufe angeben, die für dich passend sind. Wir freuen uns auf den Start!',
```

Comment:
Yet another case were it would be good to have translation on the backend :/

Reply:
That's true 😞 
It's actually the case for all the achievements.

Why don't we have any translations on the backend side?

Reply:
Not needed so far ... Maybe time to introduce it